lib/order.rb

```diff
@@ -0,0 +1,77 @@
+require "csv"
+require "awesome_print"
+
+# class that creates an instance of an order
+class Order
+
+  # creates methods that will allow reading or writing of specified instance variables
+  attr_reader :id, :products, :customer, :fulfillment_status
+  attr_writer :fulfillment_status
```

Comment:
`fulfillment_status` does not need to be an `attr_writer`.  Be a little careful when giving the user of your class the ability to change your instance variables directly like this.